.github/workflows/integration-testing.yaml

```
@@ -0,0 +1,309 @@
+                        echo "Docker image $IMAGE_TAG is not available after 10 minutes. Exiting..."
+                        exit 1
+                    fi
+                done
+
+                echo "Docker image $IMAGE_TAG is ready!"
+                
+                echo "passed=true" >> $GITHUB_OUTPUT
+                
+    build-test-stack:
+        needs: wait_for_image_ready
+        if: ${{ (always() && !cancelled()) && ( github.event_name == 'schedule'  || github.event_name == 'push' || github.event_name == 'merge_group' || (github.event_name == 'pull_request_target'  && needs.check_actor_permissions.outputs.ismaintainer == 'true')) }}
```

actionlint (1.7.12)

[error] property "check_actor_permissions" is not defined in object type {wait_for_image_ready: {outputs: {image_tag: string; mainbranch: string; namespace: string; passed: string}; result: string}}

(expression)